agent/bench-scripts/pbench-fio

```diff
@@ -596,14 +596,14 @@ function fio_run_job() {
 	if [ -n "${clients}" ]; then
 		for client in "${client_names[@]}"; do
 			local bash_c_cmd="${benchmark} --server=,${client_ports[${client}]} > client-result.txt 2> client-result.err"
-			local screen_it="screen -L -Logfile fio-server.screen.log -dmS fio-server bash -c ${bash_c_cmd}"
+			local screen_it="screen -L -Logfile fio-server.screen.log -dmS fio-server bash -c '${bash_c_cmd}'"
 			if pbench-is-local ${client}; then
 				debug_log "killing any old local fio processes and starting new a fio process for the local client"
 				killall ${benchmark} > /dev/null 2>&1
-				(cd ${benchmark_results_dir} && ${screen_it})
+				(cd ${benchmark_results_dir} && eval "${screen_it}")
```

Comment:
Is the `eval` here (and at line 606) really necessary?

That is, I don't think the addition of single quotes to line 599 changes whether `${screen_it}` can be executed directly (i.e., it shouldn't prevent `${bash_c_cmd}` from being expanded...and, if it did, then I don't think that adding an `eval` to the mix would change anything).

Reply:
I get errors without the eval: quoting through multiple levels of evaluation in shell is a nightmare.